app.py

```diff
@@ -215,6 +221,9 @@ def entrypoint():
     result = loop.run_until_complete(total_future)
     write_to_markdown(result)
 
+    if health_check_failed:
+        print('::set-output name=HEALTH_CHECK_FAILED::true')
```

Comment:
Instead of putting this in `app.py`, i would suggest putting it in the `health_check.yml::L45`

```yml
          echo "::set-output name=HEALTH_CHECK_FAILED::$(python app.py)"
```

Reply:
Thanks, addressed.